src/metadataTextTracks.js

```diff
@@ -0,0 +1,123 @@
+import videojs from 'video.js';
+
+const metadataTextTracks = {};
+let includedCues = {};
```

Comment:
As it is this variable will be shared by all players, which I don't think you want. You'll need to make this a property of something player-specific such as the videojs-contrib-ads instance. Then your methods that manipulate this object will take the player as an argument, or something like that.